insights/tests/parsers/test_rpm_v_packages.py

```diff
@@ -12,6 +12,7 @@
 ..?......    /usr/bin/sudo
 ..?......    /usr/bin/sudoreplay
 missing     /var/db/sudo/lectured (Permission denied)
+error: db5 error(-30973) from dbenv->failchk: BDB0087 DB_RUNRECOVERY: Fatal error, run database recovery
 """
```

Comment:
What about adding one more test case to have the input contain only the `error: ` lines just as the command output case mentioned in pr description part?